tests/sum.test.js

```diff
@@ -0,0 +1,5 @@
+import sum from '../sum.js';
+
+test('one plus two is three', () => {
+  expect(sum(4, 5)).toBe(9);
+});
```

Comment:
- [ ]  It would be better if you don't commit files not related to the project and added to .gitignore. Please add this `sum.test..js` file to `.gitignore` because this file doesn't test any functionality in the project. 💪 👀